api/integreatly/v1alpha1/grafananotificationchannel_types.go

```diff
@@ -0,0 +1,67 @@
+package v1alpha1
```

Comment:
Apache 2 is how this work is licensed, so consistent notice in the files is fine. Please be sure to maintain the author metadata from the original git commits.

Reply:
Hmm, I shouldn't have squashed the rebased commits then :facepalm: , I'll take a look at how to preserve the git commits

Reply:
hey @kd7lxl , how would you feel about adding yourself as the author of the rebased commit that's being pushed here? could you check out this pr, amend the commit and set the author as yourself, then force push? this would maintain your authorship status, albeit with your commits squashed into one?  If not then I can cherry-pick/rebase from your fork again later on in the week.

Reply:
you would do this with `git commit --amend --author="Author < email@email.com > "`

Reply:
I didn't write the code, so I would not be comfortable with that.

Maybe you could do `git commit --amend --author="Andrei Lavrov <alavrov@infoblox.com>"` on your squashed commit. @alavrovinfb is the one who implemented this feature on behalf of Infoblox.

Reply:
Thanks for the clarification @kd7lxl ! I'll  amend the commit and set Andrei as the author, thanks!